roottest/root/treeformula/depth/CMakeLists.txt

```diff
@@ -1,7 +1,9 @@
-#-------------------------------------------------------------------------------
-#
-# Placeholder file to translate the tests to the new CTest system. Meanwhile we
-# define a CTest test that calls 'make' in ${CMAKE_CURRENT_SOURCE_DIR}
-#
-#-------------------------------------------------------------------------------
-ROOTTEST_ADD_OLDTEST()
+ROOTTEST_ADD_TEST(create
```

Comment:
Let's remove this directory.  It is testing `TBranchElement::Branch` which was removed in 70821f21f93b65c2c6875b45c5e6c145cdd051a8.

Reply:
Yes, I remove subdirectory completely